pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -11,10 +11,13 @@
     <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
       <TargetFramework>.NETCoreApp1.0</TargetFramework>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
+      <TargetFramework>.NETCoreApp1.1</TargetFramework>
+    </ProjectReference>
   </ItemGroup>
-
+  
   <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
-    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)\netcoreapp1.0\project.json">
```

Comment:
I assume the package list is shared? If so then we shouldn't need to do this Regex hack twice. If we do need unique sets then we should just batch on the target frameworks so we don't need to duplicate the logic. 

PS: I wrote this regex hack and I really dislike it, it only exists because we didn't have a good way to have the package versions update with our update scripts. At some point we will fix that in core-setup and we can kill this hack.